RPIPs/RPIP-46.md

```diff
@@ -31,13 +31,15 @@ This RPIP is part of a set of proposals motivated by a desire to rework Rocket P
 ### Universal Adjustable Revenue Split
 This specification introduces the following pDAO protocol parameters:
 
-| Name                                               | Type       | Initial Value |
-|----------------------------------------------------|------------|---------------|
-| `node_operator_commission_share`                   | pct        | `5`           |
-| `node_operator_commission_share_council_adder`     | pct        | `0`           |
-| `voter_share`                                      | pct        | `9`           |
-| `max_node_operator_commission_share_council_adder` | pct        | `1`           |
-| `allowlisted_controllers`                          | address [] | `[]`          |
+| Name                                               | Type       | Initial Value | Guard Rails                                                                                      |
+|----------------------------------------------------|------------|---------------|--------------------------------------------------------------------------------------------------|
+| `node_operator_commission_share`                   | pct        | `5`           | reth_commission <= 100%                                                                          |
+| `node_operator_commission_share_council_adder`*    | pct        | `0`           | <= `max_node_operator_commission_share_council_adder`; <= `voter_share`; reth_commission <= 100% |
+| `voter_share`                                      | pct        | `9`           | reth_commission <= 100%                                                                          |
```

Comment:
should the reth_commission rails be `0% <= reth_commission <= 100%`?

Reply:
RP contracts only use unsigned numbers, so this isn't seen as an operative difference